documentation/experimental_features.md

```diff
@@ -37,14 +37,19 @@ Before your begin, make sure you've [updated Bolt to version 2.8.0 or
 higher](./bolt_installing.md).
 
 To get started with a Bolt project:
-1. Create a Bolt project:
-   - To create a fresh project directory, use the command `bolt project init
-   <PROJECT_NAME>`
-   - To turn an existing directory into a Bolt project directory, run `Bolt
-     project init` from the root of the directory.   
-2. Create a `bolt-project.yaml` file in the root of your Bolt project directory.
-3. Develop your Bolt plans and tasks in `plans` and `tasks` directories
-   in the root of the project directory.
+1. Create a `bolt-project.yaml` file in the root of your Bolt project directory. This can be an
+   existing directory, or a new one you make.
+1. Develop your Bolt plans and tasks in `plans` and `tasks` directories
+   in the root of the project directory, next to `bolt-project.yaml`.
+
+As of [Bolt 2.12.0](https://github.com/puppetlabs/bolt/blob/master/CHANGELOG.md#bolt-2120-2020-06-01)
+`bolt-project.yaml` contains both project configuration and [Bolt
+configuration](bolt_configuration_reference.md), excluding [transport
+configuration options](bolt_configuration_reference.md#transport-configuration-options).
+If your project contains both `bolt.yaml` and `bolt-project.yaml` files, `bolt.yaml` will be ignored
+and `bolt-project.yaml` will be preferred for loading Bolt configuration. If using
+`bolt-project.yaml`, set transport configuration in your [inventory.yaml
+file](inventory_file_v2.md).
```

Comment:
I think we should move this section down and put it under a subheading like "Setting Bolt config". Maybe under the "Whitelisting section" - depending on where the config settings would usually appear in a project file with a name and whitelist.